src/backend/marsha/core/api/video.py

```diff
@@ -1,7 +1,4 @@
 """Declare API endpoints for videos with Django RestFramework viewsets."""
```

Comment:
in the commit message
We want to control the type of media a user can **use** ... once the file **is** uploaded on S3 and can
**reject** it if the uploaded file does not have this MIME Typ